tests/run-macros/i13947/Macro_1.scala

```diff
@@ -0,0 +1,12 @@
+import scala.quoted.*
+
+inline def printTypeParams[A]: Unit = ${ printTypeParamsImpl[A] }
+
+def printTypeParamsImpl[A: Type](using q: Quotes): Expr[Unit] = {
+  import q.reflect.*
```

Comment:
```suggestion
  import quotes.reflect.*
```

Reply:
Not convinced, `quotes` can more easily be misunderstood as `scala.quotes`